src/cli.ts

```diff
@@ -155,6 +157,10 @@ type ValidateOptions = {
  * @param options.tagPrefix - The prefix used in tags before the version number.
  * @param options.fix - Whether to attempt to fix the changelog or not.
  * @param options.formatter - A custom Markdown formatter to use.
+ * @param options.versionBeforePkgRename - A version of the package before being renamed.
+ * An optional, which is required only in case of package renamed.
+ * @param options.tagPrefixBeforePkgRename - A tag prefix of the package before being renamed.
+ * An optional, which is required only in case of package renamed.
```

Comment:
```suggestion
 * @param [options.versionBeforePkgRename] - A version of the package before being renamed.
 * An optional, which is required only in case of package renamed.
 * @param [options.tagPrefixBeforePkgRename] - A tag prefix of the package before being renamed.
 * An optional, which is required only in case of package renamed.
```